ui/mirage/models/project.ts

```diff
@@ -16,7 +16,24 @@ export default Model.extend({
     result.setRemoteEnabled(this.remoteEnabled);
     result.setWaypointHcl(this.waypointHcl);
     result.setWaypointHclFormat(Project.Format.HCL);
-
+    // Somehow adding the toProtoBuf method to the variable wasn't working
+    // (probably because of the embedded relationship), so we're converting here for now.
```

Comment:
Do you mean the following didn’t work?

```ts
let varProtosList = this.variables.models.map((v) => v.toProtobuf());
```

Do you recall what the error was?

Reply:
Yeah, that didn't work. I think I remember somehow getting `toProtoBuf is not a function` (whereas when I tried to add `toProtoBufRef`, that would work)

Reply:
Could it have been a capitalization issue? I think we use `toProtobuf` rather than `toProtoBuf` in all models so far.